multy_test/test_ethereum_transaction.cpp

```diff
@@ -977,63 +977,52 @@ GTEST_TEST(EthereumTransactionTest, transfer_more_than_18ETH)
             *serialied);
 }
 
-GTEST_TEST(EthereumTransactionBuilderTest, Create_multisig_wallet_sign_bug)
+GTEST_TEST(EthereumTransactionBuilderTest, Transaction_with_signature_bug)
```

Comment:
As we are moving forward the JSON-API adoption and deprecation of the old interface, please use JSON-API for new tests on transactions.